config/settings.yml

```diff
@@ -26,3 +26,4 @@ feature_flags:
   maintenance_banner: false
   cache_courses: false
   provider_autocomplete: true
+  bursaries_and_scholarships_announced: true
```

Comment:
Shouldn this be `false`?

Reply:
So it will be tonight at midnight, but it still needs to show for the rest of the day.

After i merge this i'll put together the PR that needs to be merged at midnight

Reply:
Makes sense - not sure if its worth turning on in QA or Staging so that people can review? ...Or should we direct to the review app?

Reply:
yea good shout i'll switch it off in QA